changelogs/current.yaml

```diff
@@ -5,6 +5,9 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: tracing
+  change: |
+    Dynatrace resource detector no longer logs a warning when failing to read individual enrichment files.
```

Comment:
Actually, I think this change log is unnecessary.

Reply:
Yeah I agree. But then CI fails right? Can it be ignored?